libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -448,8 +448,9 @@ void AP_Airspeed::update_calibration(uint8_t i, float raw_pressure)
         if (state[i].cal.count == 0) {
             GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Airspeed %u unhealthy", i + 1);
         } else {
-            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Airspeed %u calibrated", i + 1);
-            param[i].offset.set_and_save(state[i].cal.sum / state[i].cal.count);
+            float offset = state[i].cal.sum / state[i].cal.count;
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Airspeed %u calibrated, offset %4.0f", i + 1, offset);
```

Comment:
I really don't want to see this information in status texts. It's spammy and a normal user doesn't need to know, and it's already in a TLOG/there for a GCS as the parameter was sent.

Reply:
I am very open to any better suggestion on helping prevent accidental skipping of covering a pitot tube during ground start....watching the GCS status messages on my TX before arming would have prevented a crash due to forgetting the pitot cover....my TX does not log TLOGs....if this is a real issue it can be turned off and on with an option I would think...

Reply:
I don't understand how adding the offset number helps you know it was calibrated. I didn't say to remove the calibrated message, I'm trying to say I don't want to see the offset number make it in here.

Reply:
because the sensor offset does not vary widely...on my 4525s its usually 84 to 125....but looking at the crash flight log it was cal'd at -380!   that made the speed estimate wildly high....unfortunately, I was having video problems immediately after takeoff and did not notice the airpseed reports > FBW_MAX....and switching to cruise to let me fiddle with the video setup, immediately crashed the plane....

Reply:
So the problem is that's not just bad airspeed cal. MS4525 does fail in flight at that level and become unusable. (I've seen multiple MS4525's fail fatally this way).

I don't want this being sent to users as it's hard to give them a reasonable understanding of what the number means, and it's just spammy. If you really want to see it on your Tx then I think you should be using scripting to do so, but for general purposes this is loading users with more spam and not improving their understanding of what they are seeing.

Reply:
I have broken this up into two PRs....this one for adding information about the cal....and another one for the LOITER only cal option...